### PR TITLE
[ MB-12368 ] Add Cypress test for Max Billable Weight updates

### PR DESCRIPTION
**Pull request by rogeruiz**
## Summary

When working on #8529, I discussed adding tests. That other PR was merged in before I could add tests. This PR is adding tests for that previous PR.

## Setup to Run Your Code

💻 You will need to use three separate terminals to test this locally.

##### Terminal 1

Start the Cypress UI locally.

```sh
make db_e2e_up e2e_test
```

## Additional Steps

1. Click on `cypress/integration/office/txo/tio-update-max-billable-weight.js` in the Cypress UI.
1. See a passing test.

### CODE REVIEW

File: cypress/integration/office/txo/tio-update-max-billable-weight.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    // TIO Payment Requests queue
+    cy.wait(['@getGHCClient', '@getPaymentRequests', '@getSortedPaymentRequests']);
+    cy.get('[data-uuid="' + paymentRequestId + '"]').click();
+
+    // Payment Requests page
+    cy.wait(['@getMovePaymentRequests']);
+
+    cy.get('#billable-weights').contains('Review weights').click();
+
+    cy.wait(['@getBillableWeight']);
```
Comment (rogeruiz) [inline cypress/integration/office/txo/tio-update-max-billable-weight.js:34]: This isn't succeeded for me locally. I'm not sure if I'm doing something wrong here. But I am encountering a error that prevents the `billable-weight` page from loading up. I borrowed most of the code to get here from the previous test that tests if **Edit max billable weight** exists on the page before it bails. So I'm not sure what's happening here. But I have to take a break from working on this for a bit today. Leaving this to communicate I'm not currently working on this.

Comment (YanZ777) [reply cypress/integration/office/txo/tio-update-max-billable-weight.js:34]: I took a look at this earlier today. It doesn't look like there's any shipments listed under the `Billable Weights` section under the `Payment Requests` tab.
The only thing that I can think of is that maybe the e2e test data is a partial flow and hasn't had any moves approved? 
What I saw in Cypress looks similar to this:
image

Comment (rogeruiz) [reply cypress/integration/office/txo/tio-update-max-billable-weight.js:34]: Oh interesting. Thank you @YanZ777

**Comment by rogeruiz** [comment]
This is the Cypress output for the test introduced here. https://app.circleci.com/pipelines/github/transcom/mymove/41782/workflows/da7b89d1-5817-4d06-b418-1ee3114c49b4/jobs/651346/parallel-runs/11?filterBy=ALL

**Comment by rogeruiz** [comment]
And it failed. I'm going to take a look at the artifacts of the test to see why it failed since it passed locally.

File: cypress/.eslintrc [hunk 2]
```diff
@@ -1,9 +1,8 @@
 {
-  "plugins": ["cypress", "prettier", "security", "no-only-tests"],
-  "extends": ["react-app", "plugin:cypress/recommended", "plugin:prettier/recommended", "plugin:security/recommended"],
+  "plugins": ["prettier", "security", "no-only-tests"],
+  "extends": ["react-app", "plugin:prettier/recommended", "plugin:security/recommended"],
   "rules": {
     "consistent-return": "error",
-    "cypress/require-data-selectors": "warn",
     "react/no-access-state-in-setstate": "error",
```
Comment (rogeruiz) [inline cypress/.eslintrc:6]: I'm removing these from the `.eslintrc` as I'm having issues on my machine around importing them. I don't see the definition in the `package.json` at the root of the project anymore. so it's safe to get rid of them.

File: cypress/integration/office/txo/tio-update-max-billable-weight.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { TIOOfficeUserType } from '../../../support/constants';
+
+describe('TIO user', () => {
+  before(() => {
+    cy.prepareOfficeApp();
+  });
+
+  beforeEach(() => {
```
Comment (rogeruiz) [inline cypress/integration/office/txo/tio-update-max-billable-weight.js:8]: This beforeEach is kind of unnecessary as I'm running a single test in here. I can remove it or leave it. So please let me know if folks think I should remove it.

**Comment by rogeruiz** [comment]
@mergifyio rebase

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm! it ran as expected in testing.

File: cypress/integration/office/txo/tio-update-max-billable-weight.js (hunk 3 above)
Comment (YanZ777) [reply cypress/integration/office/txo/tio-update-max-billable-weight.js:8]: I'd leave it!

**Pull request merged**